app/src/androidTest/java/ch/hikemate/app/ui/auth/SignInScreenTest.kt

```diff
@@ -36,8 +36,10 @@ class SignInScreenTest : TestCase() {
     mockNavigationActions = mockk(relaxed = true)
     mockAuthViewModel = mockk(relaxed = true)
 
-    // Replace the currentUser StateFlow with a mock, which is iniially null, so not signed in
+    // Replace the currentUser StateFlow with a mock, which is initially null, so not signed in
     every { mockAuthViewModel.currentUser } returns mockUserStateFlow
+    every { mockAuthViewModel.errorMessageId.value } returns null
+    every { mockAuthViewModel.loading.value } returns false
```

Comment:
Nitpick: Isn't it cleaner to return a StateFlow and not only a value? This would allow us to change the value and test for update to be correctly propagated

Reply:
Yep

Reply:
Thanks!